include/usb/usbstruct.h

```diff
@@ -28,6 +28,12 @@
  *  THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/*
+ * This header and macros below are deprecated in 2.7 release.
+ * Please replace with macros from Chapter 9 header, include/usb/usb_ch9.h
+ */
+#warning "<usb/usbstruct.h> header is deprecated, use <usb/usb_ch9.h> instead"
```

Comment:
Shouldn't this be inside of the include guard?

Reply:
> Shouldn't this be inside of the include guard?

Ping on this one

Reply:
I want to test something, I will remove or justify it at the next push.

Reply:
Moved inside include guard